go/tools/gazelle/gazelle/main.go

```diff
@@ -32,7 +32,7 @@ import (
 var (
 	goPrefix = flag.String("go_prefix", "", "go_prefix of the target workspace")
 	repoRoot = flag.String("repo_root", "", "path to a directory which corresponds to go_prefix, otherwise gazelle searches for it.")
-	mode     = flag.String("mode", "print", "print, fix or diff")
+	mode     = flag.String("mode", "fix", "print, fix or diff")
```

Comment:
yes-ish.
before https://github.com/bazelbuild/rules_go/pull/128
print prints what would be written, fix actually overwrites, diff shows the diff of the overwrite to current.

after https://github.com/bazelbuild/rules_go/pull/128
print prints what was generated, 
diff shows the diff of the overwrite to current.
while fix, will do a "merge" in place.

So I suppose 128 should be fixed to do the merge regardless, and then do one of the three operations.